pkg/nvml/nvml.go

```diff
@@ -27,37 +27,37 @@ package nvml
 import "C"
 import "unsafe"
 
-// nvmlInit_v2 function as declared in nvml/nvml.h:1732
+// nvmlInit_v2 function as declared in nvml/nvml.h:1766
```

Comment:
I wonder if there is an option in `c-for-go` to omit this info. It would make looking at the diffs much easier.

Reply:
It would seem as if this is [hard-coded](https://github.com/xlab/c-for-go/blob/73cdccf1de24f229fe86ae09fb05977ffc50dabf/generator/gen_const.go#L22), but we could strip these from the code in `pkg` with a `sed` command or something before adding?

An alternative would be a PR to `c-for-go` that adds this optionally.